.github/workflows/fitting_check.yml

```diff
@@ -0,0 +1,71 @@
+name: Validate CMS Open Data ttbar analysis
```

Comment:
This is not only checking the fitting, but the whole analysis pipeline! (for one input file)

However, it would be better if we were also to validate the fitting step for the full input data. Can you maybe add another workflow that does the statistical analysis on a preexisting set of input histograms, that where produced from the full dataset? I think we already have reference files for these histograms anyway.

Reply:
I just checked, and I dont really want to add extra workflow, since it will require extra time installation of ROOT and configuring the env. Now I just have divided validation into two steps: data downloading and fitting